aws/resource_aws_ecs_cluster.go

```diff
@@ -162,6 +200,11 @@ func resourceAwsEcsClusterRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("arn", cluster.ClusterArn)
 	d.Set("name", cluster.ClusterName)
 
+	d.Set("capacity_providers", cluster.CapacityProviders)
```

Comment:
I believe this needs to be converted to at least `[]string` via [`aws.StringValueSlice()`](https://docs.aws.amazon.com/sdk-for-go/api/aws/#StringValueSlice):

```suggestion
	if err := d.Set("capacity_providers", aws.StringValueSlice(cluster.CapacityProviders)); err != nil {
		return fmt.Errorf("error setting capacity_providers: %s", err)
	}
```